### PR TITLE
Interface prerequisites aren't limited to classes

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

**Comment by gkoz** [comment]
This doesn't achieve `pub struct TreeSortable(Object<ffi::GtkTreeSortable>): Treemodel;` yet.

**Comment by gkoz** [comment]
Apparently the same incorrect assumption was made in the `library` module

File: Gir_Gtk.toml [hunk 1]
```diff
@@ -331,6 +331,14 @@ name = "Gtk.ToolPalette"
 status = "generate"
 
 [[object]]
+name = "Gtk.TreeModel"
```
Comment (EPashkin) [inline Gir_Gtk.toml]: Can you add it to `generate` array?

**Comment by EPashkin** [comment]
Thanks.

**Comment by EPashkin** [comment]
Thanks

**Comment by EPashkin** [comment]
oops. Two `Gtk.Revealer'

**Comment by gkoz** [comment]
Hmm, this means I might have lost something.

**Comment by gkoz** [comment]
Removed the duplicate.

**Comment by EPashkin** [comment]
From old version no missed files

**Comment by EPashkin** [comment]
By way generated `Gtk.Container` contains all functions from https://github.com/gkoz/gtk-rs-gtk/blob/object_reform_gir/src/widgets/container.rs with only `get_children` differs.

**Comment by gkoz** [comment]
Yeah, I'm going to switch more classes to `generated` in that branch.

@homu r+

**Pull request merged**